project/AppDependencies.scala

```diff
@@ -17,11 +17,11 @@ object AppDependencies {
     "org.scalamock" %% "scalamock" % "5.2.0" % "test",
     "org.scalatest" %% "scalatest" % scalaTestVersion % "test, it",
     "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % "test, it",
-    "com.vladsch.flexmark" % "flexmark-all" % "0.62.2" % "test, it", // NB Added because scalatest requires it, but does not declare a dependency
-    "org.mockito" % "mockito-core" % "3.12.4"% "test, it",
+    "com.vladsch.flexmark" % "flexmark-all" % "0.64.0" % "test, it", // NB Added because scalatest requires it, but does not declare a dependency
```

Comment:
```suggestion
    "com.vladsch.flexmark" % "flexmark-all" % "0.62.2" % "test, it", // NB Added because scalatest requires it, but does not declare a dependency
```
flexmark `0.64.0` doesn't work with Java 8.

Reply:
I'm sure the tests ran and passed....  I'll recheck.